pkg/groups/managment.go

```diff
@@ -0,0 +1,8 @@
+package groups
+
+import "github.com/spf13/cobra"
+
+var ManagementCommandsGroup = &cobra.Group{
```

Comment:
I wonder if we can think of a better name...

Reply:
maybe "Core commands"?
I saw the "Management" one in another CLI tool